packages/components/src/number-control/index.tsx

```diff
@@ -52,7 +52,7 @@ function UnforwardedNumberControl(
 	forwardedRef: ForwardedRef< any >
 ) {
 	if ( hideHTMLArrows ) {
-		deprecated( 'hideHTMLArrows', {
+		deprecated( 'NumberControl hideHTMLArrows prop ', {
```

Comment:
Recently we're trying to be [even more clear](https://github.com/WordPress/gutenberg/blob/98ce5c53c9492764aeff88852a4073fb32e784ac/docs/contributors/code/backward-compatibility.md#deprecations) about the containing package:

```suggestion
		deprecated( 'wp.components.NumberControl hideHTMLArrows prop ', {
```

Reply:
Makes Sense!